include/kremlin/internal/types.h

```diff
@@ -49,19 +49,21 @@ typedef const char *Prims_string;
 
 /* The great static header headache. */
 
+#define IS_MSVC64 (defined(_MSC_VER) && defined(_M_X64) && !defined(__clang__))
+#define HAS_INT128 ((defined(__x86_64__) || defined(__x86_64) || \
+		     defined(__aarch64__) || \
```

Comment:
Can you adjust indentation here to make this a little easier to read (the `defined` is in that bracket above).